ouroboros-network/demo-connections/Main.hs

```diff
@@ -0,0 +1,143 @@
+{-# LANGUAGE ExistentialQuantification #-}
```

Comment:
This file is just a demonstration of how to use the `Connections` interface. Has nothing to do with running an ouroboros application.

Reply:
ping ;)

Reply:
Sorry, forgot about it. Will do.